### ISSUE TITLE
CLI throws exception when filename option isn't specified

### ISSUE THREAD

**Issue by atkaiser** [mintapi/mintapi#385]
**Describe the bug**
As the title states I ran into the exception below when I tried to use the CLI.

What version of MintAPI are you using? 
`1.61`
What command[s] did you run / steps to reproduce?
`python mintapi/api.py <my email>`
Stacktrace/error received:
```
mintapi % python api.py <my email>
/Users/akaiser/Documents/workspace/mintapi/mintapi/api.py:870: DeprecationWarning: Calling command line code from api.py will be deprecated in a future release.
Please call mintapi directly. For examples, see the README.md
  warnings.warn(
Traceback (most recent call last):
  File "/Users/akaiser/Documents/workspace/mintapi/mintapi/api.py", line 877, in <module>
    main()
  File "/Users/akaiser/Documents/workspace/mintapi/mintapi/cli.py", line 388, in main
    validate_file_extensions(options)
  File "/Users/akaiser/Documents/workspace/mintapi/mintapi/cli.py", line 348, in validate_file_extensions
    if not any([options.filename is None, options.filename.endswith(".json")]):
AttributeError: 'NoneType' object has no attribute 'endswith'
```

**What did you expect to happen?**
No exceptions and a JSON  of the output

**What actually happened?**
I got the above exception

**Additional context**
It seems the problem is because inside both the `any` blocks in `validate_file_extensions` a list is being passed in and I guess python needs to evaluate everything in that list before the `any` block is run so if `options.filename is None` then when it tries to evaluate the other things in the list an exception is thrown.  I plan to submit a pull request shortly.

**Comment by mike503** [comment]
this seems to still be happening to me.

ubuntu 20.04
mintapi 1.61 (from pip)
Python 3.8.10

running mintapi -h works.

mintapi with nothing gives this:

```
Traceback (most recent call last):
  File "/usr/local/bin/mintapi", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/mintapi/cli.py", line 388, in main
    validate_file_extensions(options)
  File "/usr/local/lib/python3.8/dist-packages/mintapi/cli.py", line 348, in validate_file_extensions
    if not any([options.filename is None, options.filename.endswith(".json")]):
```

when using mintapi -f something.json it does "run" but I can't actually see it doing anything so far.

however, the expected behavior would be NOT having it throw the error, and also I cannot run any of the documented examples if it doesn't include a -f with some sort of .json as the filename.

**Comment by atkaiser** [comment]
@mike503 This was fixed after 1.61, so that's why you are still getting the error on 1.61.  @mrooney would have to answer when a new version (which includes this fix) would get pushed up to PyPI.

As for why it seems to hang when you do specify `-f something.json` I don't know.
